stl/inc/limits

```diff
@@ -1116,6 +1119,20 @@ _NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val) noexcept {
 }
 #endif // _HAS_POPCNT_INTRINSICS
 
+#if defined(_M_ARM64) && !defined(_M_CEE_PURE) && !defined(__CUDACC__) && !defined(__INTEL_COMPILER) \
+    && !defined(__clang__) // TRANSITION, LLVM-51488
+#define _HAS_CNT_INTRINSICS 1
+#else // ^^^ intrinsics available ^^^ / vvv intrinsics unavailable vvv
+#define _HAS_CNT_INTRINSICS 0

```

Comment:
This should be `#undef`ed at the end of the file:

https://github.com/microsoft/STL/blob/f75c7f596c7b491168fefb5eff5164ab7ae36867/stl/inc/limits#L1149